doc/for-committers.md

```diff
@@ -88,8 +88,6 @@ This is our release checklist which can be dropped in to an issue:
 
 To build api documentation for current LT version and publish generated docs:
 
-1. In project.clj make sure that `[:codox :source-uri]` points to current LT version.
-   This step will be removed once [there is upstream support for version in :source-uri](https://github.com/weavejester/codox/issues/107)
-2. Run `script/build-api-docs.sh` on a clean git state. Make sure there are no pending git changes as this script will change git branches and push generated api docs to gh-pages.
+1. Run `script/build-api-docs.sh` on a clean git state. Make sure there are no pending git changes as this script will change git branches and push generated api docs to gh-pages.
```

Comment:
@sbauer322 I think we can just combine this item with the previous paragraph now that there's only a single item. Would you also capitalize "api" too please?